README.md

```diff
@@ -17,13 +17,14 @@ $ npm i react-vertical-timeline-component
 
 ## Usage
 
-```code|lang-jsx
+```js
```

Comment:
Did you check if it's working with catalog ?

Reply:
Sorry, I just looked at your github repository page and code was not highlighted there. Could say me what is the catalog?

Reply:
It's the tool to generate documentation at : https://stephane-monnot.github.io/react-vertical-timeline/#/

You can run it by `yarn start` command.